src/Parser/Warrior/Fury/Modules/BuffDebuff/EnrageUptime.js

```diff
@@ -8,27 +8,42 @@ import SpellLink from 'common/SpellLink';
 import { formatPercentage } from 'common/format';
 import StatisticBox from 'Main/StatisticBox';
 import Combatants from 'Parser/Core/Modules/Combatants';
+import Wrapper from 'common/Wrapper';
 
-class enrageUptime extends Analyzer {
+class EnrageUptime extends Analyzer {
   static dependencies = {
     combatants: Combatants,
   };
 
-  suggestions(when) {
-    const enrageUptime = this.combatants.selected.getBuffUptime(SPELLS.ENRAGE.id) / this.owner.fightDuration;
+  get enrageUptime() {
+    return this.combatants.selected.getBuffUptime(SPELLS.ENRAGE.id) / this.owner.fightDuration;
+  }
 
-    when(enrageUptime).isLessThan(0.70)
+  get suggestionThresholds() {
+    return {
+      actual: this.enrageUptime,
+      isLessThan: {
+        minor: 0.07,
+        average: 0.065,
+        major: 0.06,
+      },
+      style: 'percentage',
+    };
+  }
+
+  suggestions(when) {
+    when(this.suggestionThresholds.actual).isLessThan(this.suggestionThresholds.isLessThan.minor)
       .addSuggestion((suggest, actual, recommended) => {
```

Comment:
just use ``this.enrageUptime`` and in the suggestions you can define the thresholds like so 
```javascript
    const {
      isGreaterThan: {
        minor,
        average,
        major,
      },
    } = this.suggestionThresholds;
```
and then simply refer to them as ``minor``, ``average`` and ``major``